target/linux/rockchip/image/armv8.mk

```diff
@@ -2,6 +2,34 @@
 #
 # Copyright (C) 2020 Tobias Maedel
 
+define Device/friendlyarm_nanopi-m4-2gb
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi M4 2GB
+  DEVICE_DTS := rockchip/rk3399-nanopi-m4
+  SOC := rk3399
+  SUPPORTED_DEVICES :=friendlyarm,nanopi-m4
+  UBOOT_DEVICE_NAME := nanopi-m4-2gb-rk3399
```

Comment:
So, that's the only difference? Does this really justify having two otherwise identical images with different names (but which end up as the same afterwards)?